tutorials/3d/procedural_geometry/introduction.rst

```diff
@@ -0,0 +1,106 @@
+.. _doc_procedural_geometry_introduction:
+
+Introduction to procedural geometry
+===================================
+
+There are many ways to procedurally generate geometry in Godot. In this tutorial series
+we will explore a few of them. Each technique has its own benefits and drawbacks, so 
+it is best to understand each one and how it can be useful in a given situation. 
+
+What is geometry?
+-----------------
+
+Geometry is a fancy way of saying shape. In computer graphics, geometry is typically represented
+by an array of positions called "vertices". In Godot, geometry is represented by Meshes.
+
+What is a Mesh?
+---------------
+
+Many things in Godot have mesh in their name: the :ref:`Mesh <class_Mesh>`, the :ref:`ArrayMesh <class_ArrayMesh>`, 
+the :ref:`MeshInstance <class_MeshInstance>`, the :ref:`MultiMesh <class_MultiMesh>`, and 
+the :ref:`MultiMeshInstance <class_MultiMeshInstance>`. While they are all related, they have slightly different uses. 
+
+Meshes and ArrayMeshes are resources that are drawn using a MeshInstance node. Resources like 
+Meshes and ArrayMeshes cannot be added to the scene directly. A MeshInstance represents one 
+instantiation of a mesh in your scene. You can reuse a single mesh in multiple MeshInstance's 
+to draw it in different parts of your scene with different materials or transformations (scale, 
+rotation, position etc.). 
+
+If you are going to draw the same object many times, it can be helpful to use a MultiMesh with 
+a MultiMeshInstance. The MultiMeshInstance is used to draw meshes thousands of times very 
+cheaply. It takes advantage of hardware rendering in order to do so. The drawback with 
+using a MultiMeshInstance is that you are limited to one material for all instances. It uses an
+instance array to store different colors and transformations for each instance, but they all the
```

Comment:
Suggestion: Delete leftover word: "but ~~they~~ all the instances" -> "but all the instances"